src/public/publicAPIs.ts

```diff
@@ -87,8 +86,30 @@ export function initialize(hostWindow: any = window): AppInitializationEvent {
       GlobalVars.isFramelessWindow = false;
     };
   }
+}
+
+/**
+ * To notify app loaded to hide loading indicator
+ */
+export function notifyAppLoad(): void {
```

Comment:
NIT: I think `notifyAppLoaded` or `showApp` would be a better name for this API.